frontend/src/metabase/query_builder/components/dataref/DatabaseTablesPane/DatabaseTablesPane.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { ngettext, msgid } from "ttag";
+import _ from "underscore";
+import PropTypes from "prop-types";
+
+import Tables from "metabase/entities/tables";
+import Questions from "metabase/entities/questions";
+import {
+  NodeListItem,
+  NodeListItemName,
+  NodeListItemIcon,
+  NodeListTitle,
+  NodeListContainer,
+  NodeListIcon,
+  NodeListTitleText,
+} from "../NodeList.styled";
+import { ModelId } from "./DatabaseTablesPane.styled";
+
+const DatabaseTablesPane = ({ database, show, questions }) => {
```

Comment:
Is there any reason why this component is not in TypeScript?

Reply:
It wasn't in TypeScript before. Do we need to change every component to TypeScript that we touch?

Reply:
Ah, the change doesn't appear in github, so you probably thought this was a new file. It's not, I just nested it into a  directory. The original is here: https://github.com/metabase/metabase/pull/25494/files#diff-428b49b91d8f42a6f2a38f1cae18f318c4ab395519e42afdc70c255d1379c3c4